tools/generate-helpers/pg-table-bindings/store_test.go.tpl

```diff
@@ -43,9 +44,16 @@ func (s *{{$namePrefix}}StoreSuite) SetupSuite() {
 		s.T().Skip("Skip postgres store tests")
 		s.T().SkipNow()
 	}
+	{{ if .FeatureFlag }}
+	s.T().Setenv(features.{{.FeatureFlag}}.EnvVar(), "true")
+	if !features.{{.FeatureFlag}}.Enabled() {
+		s.T().Skip("Skip postgres store tests because feature flag is off")
+		s.T().SkipNow()
+	}
+	{{- end }}
 
 	s.testDB = pgtest.ForT(s.T())
-	s.store = New(s.testDB.DB)
+	s.store = CreateTableAndNewStore(context.Background(), s.testDB.DB, s.testDB.GetGormDB(s.T()))
```

Comment:
This is required because of timing

Reply:
Wait, maybe this is not needed now

Reply:
Weird, what is the timing issue? I remember all the schema are applied for store tests.